src/app/components/SubjectHeading/SubjectHeadingPageWrapper.jsx

```diff
@@ -5,8 +5,9 @@ import SubjectHeadingsContainer from './SubjectHeadingsContainer';
 import SubjectHeadingSearch from './SubjectHeadingSearch';
 
 const SubjectHeadingPageWrapper = (props) => {
-  let containerKey = props.location.query.filter || props.location.query.fromLabel || ""
-
+  const containerKey = Object.entries(props.location.query)
```

Comment:
👍good improvement

Reply:
But I am running into issues related to this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries
Scroll down to the `Polyfill` section. I needed to add that code snippet above this line

Obviously I need to figure out why my environment is different